### DISCUSSION TITLE
Can we use rainbowkit in Vue3

### DISCUSSION THREAD

**Discussion by xiaofutao** [rainbow-me/rainbowkit#1480] [Feedback]
We observed that the sample code of rainbowkit includes react, So we want to know if it can be used vue3

**Comment by DanielSinclair** [comment]
@xiaofutao Unfortunately not because Wagmi (the library RainbowKit is built around) doesn't support Vue. But we have heard some teams have luck with bundlers like Astro to mix and match Vue and React components. Let me know if you make progress in this approach! We'd love to offer an example for best practices there, and will of course support Vue directly when it is more feasible.
